.github/workflows/build-images-beta.yaml

```diff
@@ -20,6 +20,7 @@ permissions:
 jobs:
   build-and-push:
     timeout-minutes: 45
+    name: Build and Push Images
```

Comment:
Should this denote that these are beta images to differentiate from other types of images?

Reply:
This is a `job` name, the `workflow` name is already unique. It describes what the job does.
Same for the others.